test/emulator_NN.jl

```diff
@@ -0,0 +1,169 @@
+# Get ML packages
+import MLJ
+import Flux
+import MLJModels
+import MLJFlux
+Standardizer = MLJ.@load Standardizer pkg = MLJModels
+NeuralNetworkRegressor = MLJ.@load NeuralNetworkRegressor pkg = MLJFlux
+
+# Get the testing package
+import Test as TT
+
+# Get the CliMA packages
+import CloudMicrophysics as CM
+import ClimaParams as CP
+import Thermodynamics as TD
+import CloudMicrophysics.AerosolModel as AM
+import CloudMicrophysics.AerosolActivation as AA
+import CloudMicrophysics.Parameters as CMP
+
+# Container for the NN we are about to build
+struct NNBuilder <: MLJFlux.Builder
+    layer_sizes::Vector{Integer}
+    dropout::Vector
+end
+
+# Define the NN structure
+function MLJFlux.build(builder::NNBuilder, rng, n_in, n_out)
+    @assert length(builder.layer_sizes) == length(builder.dropout)
+    num_hidden_layers = length(builder.layer_sizes)
+    init = Flux.glorot_uniform(rng)
+    layers::Vector{Any} = []
+    if num_hidden_layers == 0
+        push!(layers, Flux.Dense(n_in => n_out, init = init))
+    else
+        push!(
+            layers,
+            Flux.Dense(
+                n_in => builder.layer_sizes[1],
+                Flux.sigmoid_fast,
+                init = init,
+            ),
+        )
+    end
+    for i in 1:num_hidden_layers
+        push!(layers, Flux.Dropout(builder.dropout[i]))
+        if i == num_hidden_layers
+            push!(
+                layers,
+                Flux.Dense(builder.layer_sizes[i] => n_out, init = init),
+            )
+        else
+            push!(
+                layers,
+                Flux.Dense(
+                    builder.layer_sizes[i] => builder.layer_sizes[i + 1],
+                    Flux.sigmoid_fast,
+                    init = init,
+                ),
+            )
+        end
+    end
+    return Flux.Chain(layers...)
+end
+
+# Load aerosol data reading and preprocessing functions
+include(joinpath(pkgdir(CM), "ext", "Common.jl"))
```

Comment:
I didn't realize that Common would add a lot of dependencies to the extension. Maybe it is best to have it manually included (as you are doing here) and keep its deps in the test Project.toml

Reply:
Sounds good. I'll do that, thanks!